src/domain/staking/contracts/decoders/kiln-decoder.helper.ts

```diff
@@ -0,0 +1,124 @@
+import { AbiDecoder } from '@/domain/contracts/decoders/abi-decoder.helper';
+import { LoggingService, ILoggingService } from '@/logging/logging.interface';
+import { Inject, Injectable } from '@nestjs/common';
+
+export const KilnAbi = [
+  {
+    inputs: [],
+    name: 'deposit',
+    outputs: [],
+    stateMutability: 'payable',
+    type: 'function',
+  },
+  {
+    inputs: [{ internalType: 'bytes', name: '_publicKeys', type: 'bytes' }],
+    name: 'requestValidatorsExit',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+  {
+    inputs: [{ internalType: 'bytes', name: '_publicKeys', type: 'bytes' }],
+    name: 'batchWithdrawCLFee',
+    outputs: [],
+    stateMutability: 'nonpayable',
+    type: 'function',
+  },
+] as const;
+
+export type KilnRequestValidatorsExitParameters = {
+  name: '_publicKeys';
+  type: 'bytes';
+  value: `0x${string}`;
+  valueDecoded: null;
+};
+export type KilnBatchWithdrawCLFeeParameters =
+  KilnRequestValidatorsExitParameters;
```

Comment:
I suggest we keep the decoded values as is and format it to match the backend response where it is used. What do you think?